### PR TITLE
change transcode systemd units to templated units

### PR DESCRIPTION
**Pull request by JulianJacobi**
add templated systemd units
add templated systemd targets

cleanup ansible transcoding role

### CODE REVIEW

File: ansible/roles/transcoder/templates/transcoder/lounge.sh.j2 [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+
+if [[  $# -ne 1 ]] ; then
+    echo "Please give exactly one parameter: name of the streaming endpoint to transcode"
+    exit 1;
+fi
+
+#Lounge passwords
+{% for stream_key in transcoder_lounges %}
+PASSWORD_{{ stream_key }}="{{ lookup('keepass', 'ansible/audio-pi/icecast_'+ stream_key +'.password') }}"
+{% endfor %}
```
Comment (MaZderMind) [inline ansible/roles/transcoder/templates/transcoder/lounge.sh.j2]: That could be a bash-array. But on the other hand – bash arrays 🤮

File: ansible/roles/transcoder/templates/transcoder/lounge.sh.j2 [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
+
+PASSWORD_VARIABLE=PASSWORD_$1
```
Comment (MaZderMind) [inline ansible/roles/transcoder/templates/transcoder/lounge.sh.j2]: It's obvious that this is a variable; the name should just be PSSSWORD

File: ansible/roles/transcoder/templates/transcoder/lounge.sh.j2 [hunk 3]
```diff
@@ -1,18 +1,30 @@
...
 	\
 	-map 0:a:0 \
 		-metadata:s:a:0 title="Lounge MP3" \
 		-c:a:0 libmp3lame -b:a:0 96k \
 	\
 	-f mp3 \
 	-content_type audio/mp3 \
-	-password "{{ lookup('keepass', 'ansible/audio-pi/icecast_{{ item }}.password') }}" \
-	icecast://{{ transcoder.lounge_push_endpoint }}/{{ item }}.mp3
+	-password "${!PASSWORD_VARIABLE}" \
```
Comment (MaZderMind) [inline ansible/roles/transcoder/templates/transcoder/lounge.sh.j2]: Was macht das `!` da?

**Comment by iSchluff** [comment]
Merged into events/36c3

**Pull request closed without merging**